### PR TITLE
headerComponent

### PR DESCRIPTION
**Pull request by naremanhilles**
commonComponent/create common component header and styling it #103

relates #93

### CODE REVIEW

File: client/src/Components/CommonComponent/Header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// eslint-disable-next-line react/prop-types
+export default function Header({ title, icon }) {
+  return (
+    <div className="header">
+      {icon}
+      <h2>{title}</h2>
+      <hr className="line" />
+    </div>
+  );
+}
```
Comment (engshorouq) [inline client/src/Components/CommonComponent/Header/index.jsx]: I expect the Eslint give you an error say `title is missing in props validation` and also for `icon is missing in props validation`
don't forget to add validation for it see this link https://reactjs.org/docs/typechecking-with-proptypes.html

File: client/src/Components/CommonComponent/Header/index.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+// eslint-disable-next-line import/no-unresolved
+import './style.css';
+
+// eslint-disable-next-line react/prop-types
+export default function Header({ title, icon }) {
```
Comment (hshahwan) [inline client/src/Components/CommonComponent/Header/index.jsx]: Keep your code consistent with ES6

File: client/package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "proxy": "http://localhost:4400",
   "dependencies": {
+    "antd": "^3.19.1",
```
Comment (hshahwan) [inline client/package.json]: why you install this package and you don't use it?

Comment (naremanhilles) [reply client/package.json]: Used by trial time.

File: client/src/Components/CommonComponent/Header/index.jsx (hunk 2 above)
Comment (naremanhilles) [reply client/src/Components/CommonComponent/Header/index.jsx]: > Keep your code consistent with ES6

fixed :)

File: client/src/Components/CommonComponent/Header/style.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
+.header i{
```
Comment (hshahwan) [inline client/src/Components/CommonComponent/Header/style.css]: you don't have i tage, I know you mentioned your icon, but to relate it

File: client/src/Components/CommonComponent/Header/style.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  display: inline;
+  float: right;
+  margin-right: 2rem;
+  font-size: 2.5rem;
+}
+.header h2{
```
Comment (engshorouq) [inline client/src/Components/CommonComponent/Header/style.css]: use BEM the class name like that `header__h2`

File: client/src/Components/CommonComponent/Header/style.css (hunk 4 above)
Comment (engshorouq) [inline client/src/Components/CommonComponent/Header/style.css]: also, use BEM the class name like that `header__icon`  and write this commit on the above of file to know any one use this file `/* when pass Icon put insilde it <Icon type="..." className="header__icon" /> */`

File: client/src/Components/CommonComponent/Header/style.css (hunk 5 above)
Comment (naremanhilles) [reply client/src/Components/CommonComponent/Header/style.css]: thanks shorouq :)

File: client/src/Components/CommonComponent/Header/style.css (hunk 4 above)
Comment (naremanhilles) [reply client/src/Components/CommonComponent/Header/style.css]: ok idit it thanks shorouq :+1:

**Pull request merged**